tests/test_basic.py

```diff
@@ -4,220 +4,192 @@
 
 def test_gpkg_bbox(script_runner):
-    ret = script_runner.run('python3', 'geoextent',
+    ret = script_runner.run('python', 'geoextent',
         '--path', 'Testdata/census2016_cca_qld_short.gpkg', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+    assert "[-43.7405, 96.8169, -9.14218, 167.998]" in ret.stdout, "bbox is printed to console"
 
 def test_csv_bbox(script_runner, tmpdir):
-    ret = script_runner.run('python3', 'geoextent',
-        '--path', 'Testdata/cities_NL.csv', 
+    ret = script_runner.run('python', 'geoextent',
+        '--path', 'Testdata/folder/cities_NL.csv', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[[10,20],[11,50]" in ret.stdout, "bbox is printed to console"
+    assert "[6.574722, 51.434444, 4.3175, 53.217222]" in ret.stdout, "bbox is printed to console"
 
 def test_csv_time(script_runner, tmpdir):
-    ret = script_runner.run('python3', 'geoextent',
-        '--path', 'Testdata/cities_NL.csv', 
+    ret = script_runner.run('python', 'geoextent',
+        '--path', 'Testdata/folder/cities_NL.csv', 
         '--detail', 'bbox',
         '--time')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "temporal extent is from 2017 ==> 2018" in ret.stdout, "time value is printed to console"
+    assert "[2018-09-30]" in ret.stdout, "time value is printed to console"
 
 def test_csv_time_invalid(script_runner, tmpdir):
-    ret = script_runner.run('python3', 'geoextent',
-        '--path', 'Testdata/cities_NL.csv', 
+    ret = script_runner.run('python', 'geoextent',
+        '--path', 'Testdata/folder/cities_NL.csv', 
         '--detail', 'bbox',
         '--time')
-    #print(ret.stdout)
     assert not ret.success, "process should return success"
     assert ret.stderr == 'invalid time format', "stderr should not be empty"
 
 def test_gml_bbox(script_runner):
-    ret = script_runner.run('python3', 'geoextent',
+    ret = script_runner.run('python', 'geoextent',
         '--path', 'Testdata/clc_1000_PT.gml', 
         '--detail', 'bbox')
-    #print(ret.stdout)
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
-    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+    assert "[-17.5420724159224, 32.3966928193202, -6.95938792923511, 39.3011352746141]" in ret.stdout, "bbox is printed to console"
```

Comment:
Very good you have the correct bounding boxes now (just commenting on this one). The only thing we have to watch out for are small changes in the floating point numbers... let's see if those come up and solve them then.